Projects/Parking Operator/main.py

```diff
@@ -0,0 +1,139 @@
+from models import CarPark, Car
+from helpers import clear_display
+from datetime import datetime
+
+mycarpark = CarPark("MyCarpark", 5)
+
+choice = ''
+
+while choice != '9': 
+    
+    print(f"\nWelcome to {mycarpark.name}! Please pick a task below: ")
+    print("\n1. Display all cars.")
+    print("2. Add a car.")
+    print("3. Pay for parking.")
+    print("4. Remove car.")
+    print("5. Display history")
+    print("9. Quit program.")
+
+    
+    choice = input("\nPlease pick a task above: ")
+
+    if choice == '1': 
+        clear_display()
+        if mycarpark.is_carpark_empty(): 
+            print(f"\nWarning: {mycarpark.name} is empty!")
+            continue
+
+
+        mycarpark.display_car_collection()
+
+    elif choice == '2':
+
+        if mycarpark.is_full(): 
+            
+            clear_display()
```

Comment:
instead of using the clear_display() function you could import this in main and use
```python
from os import system
system("cls")
```